release/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jira==3.8.0
```

Comment:
It looks like we need the Apache license in this file otherwise `./gradlew rat` fails.